packages/orchestrator/lib/clients/client.integration.test.ts

```diff
@@ -34,9 +34,34 @@ describe('OrchestratorClient', async () => {
         scheduler.stop();
         await dbClient.clearDatabase();
     });
+
+    describe('recurring schedule', () => {
+        it('should be created', async () => {
+            const res = await client.recurring({
+                name: 'Task',
+                startsAt: new Date(),
+                frequencyMs: 300_000,
+                args: {
+                    type: 'sync',
+                    syncId: 'sync-a',
+                    syncName: rndStr(),
+                    syncJobId: 5678,
+                    connection: {
+                        id: 123,
+                        connection_id: 'C',
+                        provider_config_key: 'P',
+                        environment_id: 456
+                    },
+                    debug: false
+                }
+            });
+            expect(res.isOk()).toBe(true);
+        });
+    });
+
     describe('heartbeat', () => {
         it('should be successful', async () => {
-            const scheduledTask = await client.schedule({
+            const scheduledTask = await client.immediate({
```

Comment:
I was originally planning to have a single function to create immediate tasks and recurring schedules but creating a schedule doesn't necessary mean scheduling a task so the return type is different. 
There is now two methods: `immediate` to schedule a task immediately and `recurring` that is creating a schedule